gradle/plugins/src/main/kotlin/com/hedera/gradle/tasks/GitClone.kt

```diff
@@ -59,7 +59,7 @@ abstract class GitClone : DefaultTask() {
                     commandLine(
                         "git",
                         "clone",
-                        "https://github.com/hashgraph/hedera-protobufs.git",
+                        "https://github.com/LimeChain/hedera-protobufs.git",
                         "-q"
```

Comment:
This will be changed, once https://github.com/hashgraph/hedera-protobufs/pull/371 is merged.